### PR TITLE
Add large/small card 

### PR DESCRIPTION
**Pull request by mawenyuan1997**
Add large card and small card components. Change card style on homepage.

### CODE REVIEW

File: _sass/components/_card.scss [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    font-size: 24px;
+    line-height: 36px;
+    flex: none;
+    order: 0;
+    align-self: stretch;
+    flex-grow: 0;
+  }
+  .small-card-description{
+    padding: 0 30px;
+//     width: 312px;
```
Comment (tslmy) [inline _sass/components/_card.scss]:
```suggestion
```

File: _sass/components/_card.scss [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    order: 1;
+    align-self: stretch;
+    align-items: center;
+    flex-grow: 0;
+  }
+  .large-card-link {
+    box-sizing: border-box;
+    flex-direction: row;
+    padding: 12px 24px;
+//     width: 104px;
```
Comment (tslmy) [inline _sass/components/_card.scss]:
```suggestion
```

File: _sass/components/_card.scss [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    font-size: 24px;
+    line-height: 36px;
+    flex: none;
+    order: 0;
+    align-self: stretch;
+    flex-grow: 0;
+  }
+  .large-card-description{
+    padding: 0 30px;
+//     width: 312px;
```
Comment (tslmy) [inline _sass/components/_card.scss]:
```suggestion
```

File: _sass/components/_card.scss [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    padding-bottom: 20px;
+    list-style-position: inside;
+    box-sizing: border-box;
+
+    font-family: 'Noto Sans SC';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #000000;
```
Comment (tslmy) [inline _sass/components/_card.scss]:
```suggestion
    color: $black;
```

File: _sass/components/_card.scss [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+    flex-grow: 0;
+    font-family: 'Noto Sans SC';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 22px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #AF1D25;
```
Comment (tslmy) [inline _sass/components/_card.scss]: Is this a defined color? If so, let's use the variable.

File: _sass/components/_card.scss [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 36px;
+    flex: none;
+    order: 0;
+    align-self: stretch;
+    flex-grow: 0;
+  }
+  .large-card-description{
```
Comment (tslmy) [inline _sass/components/_card.scss]:
```suggestion
  .large-card-description {
```

File: _sass/components/_card.scss [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+  background-color: $white-offset;
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  width: 360px;
+  height: 496px;
+  left: 16px;
+  top: 16px;
+  align-items: flex-start;
+  padding: 0px;
```
Comment (tslmy) [inline _sass/components/_card.scss]: Can we extract rules that are common across `.large-card` and `.small-card` into a `.card`?

**Review by tslmy — CHANGES_REQUESTED** [review]

File: _sass/components/_card.scss (hunk 7 above)
Comment (mawenyuan1997) [reply _sass/components/_card.scss]: Yeah just extracted common rules

**Comment by tslmy** [comment]
Included in another PR.

**Pull request closed without merging**